cmd/list/dnsdomains/cmd.go

```diff
@@ -54,6 +56,13 @@ func init() {
 		"List all DNS domains (default lists just user defined).",
 	)
 
+	flags.BoolVar(
+		&args.hostedCp,
+		"hosted-cp",
+		false,
+		"If listing dns-domains used for Hosted Control Plane clusters",
```

Comment:
@hunterkepley Minor suggestion: Make this a statement of what the flag does e.g "Filter to list only DNS Domain used for Hosted Control Plane clusters" or something similar.

Minor, so feel free to ignore if you disagree or it's this way for a reason.

Reply:
I'm pretty bad at descriptions like this, I like yours better 🙂